app/models/spree/user.rb

```diff
@@ -168,6 +168,7 @@ def set_login
     def self.friendly_token
       SecureRandom.base64(15).tr('+/=', '-_ ').strip.delete("\n")
     end
+    private_class_method :friendly_token
```

Comment:
I did a search, and it looks like this method is no longer used since e4d307fe5efe88357b9a1e83796cf2941a7a3fdb

Therefore, I think we can remove it.

```suggestion
```